raiden/api/rest.py

```diff
@@ -1650,6 +1652,32 @@ def settlement_light(self,
             )
             return result
 
+        message = LightClientMessageHandler.get_light_client_protocol_message_by_internal_identifier(
+            internal_msg_identifier=internal_msg_identifier,
+            wal=self.raiden_api.raiden.wal
+        )
+
+        if not message:
+            return ApiErrorBuilder.build_and_log_error(
+                errors="Light Client Message with internal_msg_identifier = {} not found"
+                       .format(internal_msg_identifier),
+                status_code=HTTPStatus.BAD_REQUEST,
+                log=log
+            )
+
+        if message.is_signed:
+            return ApiErrorBuilder.build_and_log_error(
+                errors="Failed trying to settle a channel that's already settled",
+                status_code=HTTPStatus.CONFLICT,
+                log=log
+            )
+
+        LightClientMessageHandler.update_onchain_light_client_protocol_message_set_signed_transaction(
```

Comment:
we are ignoring the returned value here

Reply:
because we don't need it, in case of some update error this operation will repeat, but if this fails probably the database is not working and the lumino node shouldn't be working neither

Reply:
got it, thanks.